### PR TITLE
Resize video while resizing

### LINKED ISSUE #755: Video not resizing together with the selection box while resizing
## Bug Description

Weird visual when resizing video. 

## Expected behavior

Video should resize while resizing.

## Steps to Reproduce

Resize video 

## Screenshots

Screenshot from 2020-03-25 12-25-38

### PR DESCRIPTION
**Pull request by miina**
Fixes #755

### CODE REVIEW

File: assets/src/edit-story/elements/image/display.js [hunk 1]
```diff
@@ -24,26 +24,20 @@ import { useRef } from 'react';
...
   ${mediaWithScale}
 `;
 
-function ImageDisplay({
-  element: { id, resource, scale, focalX, focalY },
-  box: { width, height },
-}) {
-  const imageRef = useRef(null);
+function ImageDisplay({ element, box }) {
+  const { resource } = element;
```
Comment (spacedmonkey) [inline assets/src/edit-story/elements/image/display.js]:
```suggestion
  const { resource, scale, focalX, focalY } = element;
```

File: assets/src/edit-story/elements/image/display.js [hunk 2]
```diff
@@ -24,26 +24,20 @@ import { useRef } from 'react';
...
+  const { scale, focalX, focalY } = element;
```
Comment (spacedmonkey) [inline assets/src/edit-story/elements/image/display.js]:
```suggestion
```

**Review by spacedmonkey — APPROVED** [review]
One minor nit pick, but this looks good.

**Comment by swissspidy** [comment]
Added to alpha branch

**Pull request merged**